src/components/include/security_manager/crypto_manager.h

```diff
@@ -66,7 +67,8 @@ class CryptoManager : public policy::PolicyHandlerObserver {
   virtual void ReleaseSSLContext(SSLContext* context) = 0;
   virtual std::string LastError() const = 0;
 
-  virtual bool IsCertificateUpdateRequired() const = 0;
+  virtual bool IsCertificateUpdateRequired(
```

Comment:
@AByzhynar description

Reply:
@LuxoftAKutsan Changed in eb15dd852a0ea418bfa444b74c8f15d792e254ad